src/js/compositions.js

```diff
@@ -57,13 +49,22 @@ trace = function( msg ) {
     }
 }
 
+//extrapolating function for create div and add className
+create = function(tagName, className, container) {
+    var el = document.createElement(tagName);
+    el.className = className;
+    if (container) {
+        container.appendChild(el);
+    }
+    return el;
+}
+
 KL.Pullquote = (function() {
 
     // DOM ELEMENTS
     this.el = {
-        container: KL.Dom.get("pullquote-container"),
+        container: document.getElementById("pullquote-container"),
```

Comment:
I definitely think this wins for clarity, and there are no cross-browser issues with `getElementById` anymore so big 👍  to this